test/system/600-completion.bats

```diff
@@ -274,6 +291,9 @@ function _check_completion_end() {
     check_shell_completion
 
     # cleanup
+    run_podman secret rm $random_secret_name
+    rm -f $secret_file
```

Comment:
FYI for next time: cleanup isn't necessary here, because `$PODMAN_TMPDIR` is new for each test and wiped in teardown. My purpose was to make each test line be important, so the test writer/maintainer wouldn't have to worry about clutter.